packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -104,7 +108,7 @@ export async function sendSignedXml({
       initialDelay: initialDelay.asMilliseconds(),
       maxAttempts: isDq ? 4 : 3,
       //TODO: This introduces retry on timeout without needing to specify the http Code: https://github.com/metriport/metriport/pull/2285. Remove once PR is merged
-      httpCodesToRetry: ["ECONNREFUSED", "ECONNRESET", "ETIMEDOUT", "ECONNABORTED"],
+      httpCodesToRetry: ["ECONNREFUSED", "ECONNRESET", "ETIMEDOUT"],
```

Comment:
rollback

Reply:
I suggest leaving it - discussed on Slack